app/assets/stylesheets/custom.css

```diff
@@ -127,8 +158,3 @@ th{
 td{
   font-size: 20px;
 }
-
-.col-centered{
```

Comment:
I actually need this rule for the jokes index view and maybe later on. Is there somewhere else that this is ruining the styling?